Backpack/BarChart/Classes/BarChartCollectionViewDataSource.swift

```diff
@@ -0,0 +1,38 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2018-2020 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import UIKit
+
+@objc(BPKBarChartCollectionViewDataSource)
+public protocol BPKBarChartCollectionViewDataSource {
+
+    /// The number of bars to display in the chart
+    func numberOfBarsInChart(barChartCollectionView: BPKBarChartCollectionView) -> Int
+
+    /// The title to show underneath a specific bar
+    func titleForBar(barChartCollectionView: BPKBarChartCollectionView, atIndex: Int) -> String
+
+    /// The subtitle to show under the title of a specific bar
+    func subtitleForBar(barChartCollectionView: BPKBarChartCollectionView, atIndex: Int) -> String
```

Comment:
This should be optional right? You don't have to provide a subtitle after all

Reply:
For now, I assumed you would for simplicity. I guess we can relax it in the future if there's a use case?

Reply:
Alright